workspaces/scorecard/plugins/scorecard-node/report.api.md

```diff
@@ -0,0 +1,37 @@
+## API Report File for "@red-hat-developer-hub/backstage-plugin-scorecard-node"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { ExtensionPoint } from '@backstage/backend-plugin-api';
+import { JsonValue } from '@backstage/types/index';
+import { Metric } from '@red-hat-developer-hub/backstage-plugin-scorecard-common';
+import { MetricType } from '@red-hat-developer-hub/backstage-plugin-scorecard-common';
+import { MetricValue } from '@red-hat-developer-hub/backstage-plugin-scorecard-common';
+
+// @public (undocumented)
+export interface MetricProvider<
+  T extends MetricType = MetricType,
+  TOptions extends JsonValue = JsonValue,
+> {
+  // (undocumented)
+  calculateMetric(options?: TOptions): Promise<MetricValue<T>>;
+  // (undocumented)
+  getMetric(): Metric<T>;
+  // (undocumented)
+  getProviderDatasourceId(): string;
+  // (undocumented)
+  getProviderId(): string;
+}
+
+// @public (undocumented)
+export interface ScorecardMetricsExtensionPoint {
+  // (undocumented)
+  addMetricProvider(
+    ...metricProviders: Array<MetricProvider | Array<MetricProvider>>
```

Comment:
Why do we need Array<Array<MetricProvider>> ?
 
 Maybe it will be easier just to have  `...metricProviders: Array<MetricProvider>`?